diff --git a/SZ/config/cmake/H5SZMacros.cmake b/SZ/config/cmake/H5SZMacros.cmake
index f0c1ecf5..11f959bc 100644
--- a/SZ/config/cmake/H5SZMacros.cmake
+++ b/SZ/config/cmake/H5SZMacros.cmake
@@ -48,7 +48,7 @@ macro (EXTERNAL_SZ_LIBRARY compress_type libtype)
             -DBUILD_SZ_EXAMPLES:BOOL=OFF
             -DSZ_PACKAGE_EXT:STRING=${SZ_PACKAGE_EXT}
             -DSZ_EXTERNALLY_CONFIGURED:BOOL=OFF
-            -DSZ_FIND_DEPS:BOOL=OFF
+            -DSZ_FIND_DEPS:BOOL=ON
             -DCMAKE_BUILD_TYPE:STRING=${CMAKE_BUILD_TYPE}
             -DCMAKE_INSTALL_PREFIX:PATH=${CMAKE_INSTALL_PREFIX}
             -DCMAKE_RUNTIME_OUTPUT_DIRECTORY:PATH=${CMAKE_RUNTIME_OUTPUT_DIRECTORY}
diff --git a/config/cmake/HDFPluginMacros.cmake b/config/cmake/HDFPluginMacros.cmake
index 6cb9e043..164a13cf 100644
--- a/config/cmake/HDFPluginMacros.cmake
+++ b/config/cmake/HDFPluginMacros.cmake
@@ -280,8 +280,8 @@ macro (HDF5_SUPPORT link_hdf)
             #plugin source needs to be linked with HDF5
             set (LINK_LIBS ${LINK_LIBS} ${HDF5_C_STATIC_LIBRARY})
           endif ()
-          set_property (TARGET ${HDF5_NAMESPACE}h5dump PROPERTY IMPORTED_LOCATION "${HDF5_TOOLS_DIR}/h5dump")
-          set_property (TARGET ${HDF5_NAMESPACE}h5repack PROPERTY IMPORTED_LOCATION "${HDF5_TOOLS_DIR}/h5repack")
+          # set_property (TARGET ${HDF5_NAMESPACE}h5dump PROPERTY IMPORTED_LOCATION "${HDF5_TOOLS_DIR}/h5dump.js")
+          # set_property (TARGET ${HDF5_NAMESPACE}h5repack PROPERTY IMPORTED_LOCATION "${HDF5_TOOLS_DIR}/h5repack.js")
         endif ()
       endif ()
     else ()
