diff --git a/CMakeLists.txt b/CMakeLists.txt
index c915b0ab..ad11406f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -139,6 +139,8 @@ if (NOT MINGW)
     endif ()
   elseif (WIN32 AND NOT CMAKE_C_COMPILER_ID MATCHES "Intel[Ll][Ll][Vv][Mm]")
     FILTER_OPTION (LZ4)
+  elseif (EMSCRIPTEN)
+    FILTER_OPTION (LZ4)
   else ()
     set (ENABLE_LZ4 OFF CACHE BOOL "" FORCE)
   endif ()
diff --git a/SZ/config/cmake/H5SZMacros.cmake b/SZ/config/cmake/H5SZMacros.cmake
index aa133a0d..3de93818 100644
--- a/SZ/config/cmake/H5SZMacros.cmake
+++ b/SZ/config/cmake/H5SZMacros.cmake
@@ -12,7 +12,7 @@
 
 include (FetchContent)
 #-------------------------------------------------------------------------------
-macro (EXTERNAL_SZ_LIBRARY compress_type libtype)
+macro (EXTERNAL_SZ_LIBRARY compress_type)
   if (${compress_type} MATCHES "GIT")
     FetchContent_Declare (SZF
         GIT_REPOSITORY ${SZ_URL}
